Docs: Add Installation YouTube Videos to documentation pages. (#88443)
grafana/grafana: docs/sources/setup-grafana/installation/debian/index.md

If you install via the `.deb` package or `.tar.gz` file, then you must manually update Grafana for each new version.
{{% /admonition %}}

> The following video guides you through the steps, common commands and the ways in which you can install Grafana in Debian and Ubuntu as described in this document:

Reviewer: ```suggestion
The following video guides you through the steps, common commands, and how to install Grafana in Debian and Ubuntu as described in this document:
```
